### ISSUE TITLE
[$10][MSFT-73] [Member] Null or empty alternative text in Feature Home

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2787]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2. Inspect Element
3. Observe the alternative text in User Profile Icon.

**Expected Result**
Null or empty alternative text in Feature Home should not be present.

**Actual Result**
Null or empty alternative text in Feature Home is shown as below:
<img src="https://www.topcoder.com/wp-content/uploads/2018/05/Homepage.png
https://www.topcoder.com/wp-content/uploads/2018/05/Homepage.png
" alt="" class="featured-home-img">

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non-Text Content
**Screen Reader Used**: 
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=ae14bf9a%2D590b%2D4bb4%2Da40a%2D4c6822c880a4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%2073

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096279 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096279 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096279 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@codeMinter  then what alt text is to be displayed?

**Comment by PrakashDurlabhji** [comment]
@veshu is this home page in scope?

**Comment by veshu** [comment]
@PrakashDurlabhji No it is out of scope for now.

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097856 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
